runtime/queries/svelte/indents.toml

```diff
@@ -0,0 +1,18 @@
+indent = [
+  "element"
+  "if_statement"
+  "each_statement"
+  "await_statement"
+]
+
+branch = [
```

Comment:
There is no branch or ignore keys, you might want to add the closing tag as outdent instead

Reply:
changed